ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/StructHelper.java

```diff
@@ -45,6 +45,23 @@ public static String getNameUnderscores(final String name) {
         return Helper.camelToUnderscore(Helper.removeTrailingRequestHandler(Helper.unqualifiedName(name)));
     }
 
+    /**
+     * Need to special case underscores to camel-case conversion due to inconsistent capitalization of ID
+     * @param structMemberName
+     * @return camel-cased structMemberName
+     */
+    public static String convertStructMemberNameToCamelCase(final String structName, final String structMemberName) {
+        if (structMemberName.equals("id")) {
+            if (structName.equals("ds3_user_response")) {
+                return "Id";
+            }
+            return "ID";
+        }
+        //return Helper.underscoreToCamel(structMemberName);
```

Comment:
I removed this whole funtion in the next pull request when I add the nameToMarshall property to the StructMember